board/batocera/fsoverlay/etc/init.d/S11share

```diff
@@ -169,16 +169,16 @@ mountNetwork() {
     # execute all commands in /boot/batocera-boot.conf which are like : sharenetwork_cmd1=my command
     if ! grep -E '^[ ]*sharenetwork_[a-z]*[0-9][ ]*=' "${SHARECONFFILE}" |
 	    sed -e s+'^[ ]*sharenetwork_\([a-z]*\)[0-9][ ]*='+'\1 '+ |
-	    while read CTYPE CMD
+	    while read -r CTYPE CMD
 	    do
 		XTRY=5  # X tries and give up
 		XWAIT=4 # N seconds between each try
 
 		while test "${XTRY}" -gt 0
 		do
-		    let XTRY--
+		    (( XTRY-- ))
 
-		    CMD_EXEC=echo
+		    CMD_EXEC=$(echo)
```

Comment:
No... this can possible break scripts. You force a subshell with `$(echo)` this command and possible added arguments will then become program calls which will break the script (not in this script but that is just pure luck). You can try to remove this variable here. It's rebuild later in the script.

Reply:
revert